Pomodoro.Core/Models/Statistics/AnalyticsPerHour.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Pomodoro.Core.Models.Statistics
+{
+    public class AnalyticsPerHour
+    {
+        public int Hour { get; set; }
+        public int PomodorosDone { get; set; }
+        public float TimeSpent { get; set; }
```

Comment:
maybe better to use DateTime here, or are we storing only hours value?

Reply:
I meant to store only the hour value here since the date is known to the `DailyStatistics` object which stores a list of `AnalyticsPerHour` objects

Reply:
> I meant to store only the hour value here since the date is known to the `DailyStatistics` object which stores a list of `AnalyticsPerHour` objects

okay, no objections then  :)